other/upgrade_2-1_mysql.sql

```diff
@@ -2647,7 +2647,7 @@ if (!array_key_exists($modSettings['smiley_sets_default'], $filtered))
 
 ---# Drop obsolete filename column from smileys table
 ALTER TABLE {$db_prefix}smileys
-DROP COLUMN filename;
+DROP COLUMN IF EXISTS filename;
```

Comment:
Mariadb support this with 10.0 https://stackoverflow.com/questions/173814/using-alter-to-drop-a-column-if-it-exists-in-mysql
and mysql 8.0 not even now -> don't do it in mysql upgrade

Reply:
Thanks.